### PR TITLE
Adding geoserver coastalines layers

### PR DESCRIPTION
**Pull request by Indiphile**

### CODE REVIEW

**Comment by SeffatChowdhury** [comment]
@Indiphile can you also please request a review from @fangfy

File: services/ows_refactored/prod_af_ows_root_cfg.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "allowed_urls": [
             "https://ows.digitalearth.africa",
             "https://ows-latest.digitalearth.africa",
+            "https://geoserver.digitalearth.africa/geoserver/web",
```
Comment (SeffatChowdhury) [inline services/ows_refactored/prod_af_ows_root_cfg.py:28]: Please remove the "/geoserver/web"

**Review by SeffatChowdhury — CHANGES_REQUESTED** [review]

File: services/ows_refactored/prod_af_ows_root_cfg.py (hunk 1 above)
Comment (Indiphile) [reply services/ows_refactored/prod_af_ows_root_cfg.py]: Yes, thank you @SeffatChowdhury I removed the line.

**Review by fangfy — COMMENTED** [review]
Thanks Indiphile. I think while this works, it's possible to include some configuration in the url for better styling. And we should also think about where in the catalog to include the coastline. @robbibt has a good example somewhere that should be helpful.

**Comment by robbibt** [comment]
Hi all, here is a Terria config for Coastlines that I put together a few months ago: https://gist.github.com/robbibt/cf6406d84fa2a4f1b619d8820ee0ff41

It essentially keeps the existing DE Africa Maps config the same, but inserts the new Coastlines layer into a new folder called `Coastal`. This keeps the directory structure nice and clean/simple, and means you don't need seperate OWS and Geoserver directories. You can see a preview of what this looks like here: 

https://maps.digitalearth.africa/#clean&https://gist.githubusercontent.com/robbibt/cf6406d84fa2a4f1b619d8820ee0ff41/raw/402ac0a114f3d4b2b4fa50ba8584bb1f2b94fa25/africa-prod.json

image

To implement this, you can simply replace the existing `terria/africa-prod.json` file with that config file.

**Comment by fangfy** [comment]
@Indiphile this PR is superceded and should be closed?

**Comment by Indiphile** [comment]
> @Indiphile this PR is superceded and should be closed?
It's closed now.

**Pull request closed without merging**